privacyidea/lib/tokens/u2ftoken.py

```diff
@@ -323,13 +322,17 @@ def update(self, param, reset_failcount=True):
         :return: None
         """
         TokenClass.update(self, param)
-        description = "U2F initialization"
+        automatic_description = "U2F initialization"
```

Comment:
i think we don't need a default here, `automatic_description` will always be overridden by `parse_registration_data()` (wither with `""` or with the CN).

Reply:
You mean from the point of the program flow?
I will check this.

Reply:
Done in bb978a447947f51dc645c265c2f8f229ca60ee5a